cookbooks/bcpc/recipes/cronjobs.rb

```diff
@@ -18,9 +18,12 @@
 
 # Base cronjobs/ pseudo-cronjobs that should be on all machines in the cluster.
 
+polyinstantion_dir = node['bcpc']['pam_namespace']['polyinstantion_dir']
+shm_polyinstantion_dir = node['bcpc']['pam_namespace']['shm_polyinstantion_dir']
+
 clear_tmp = node['bcpc']['cronjobs']['clear_tmp']
-execute 'clear /tmp' do
-  command '/usr/bin/find /tmp -type f '\
+execute 'clear tmp dirs' do
+  command '/usr/bin/find /tmp #{polyinstantion_dir} /dev/shm/#{shm_polyinstantion_dir} -type f '\
           "-atime +#{clear_tmp['atime_age']} -delete && "\
```

Comment:
`/tmp` is usually `tmpfs`, but we should verify this is not mounted with `noatime`, as this has likely been broken forever if so.

Reply:
On a VM cluster I see that root has atime enabled and /tmp is on root:
```ubuntu@tmpdirs-bcpc-vm4:~$ cat /etc/mtab 
/dev/mapper/tmpdirs--bcpc--vm4--vg-root / ext4 rw,errors=remount-ro 0 0
proc /proc proc rw,noexec,nosuid,nodev 0 0
sysfs /sys sysfs rw,noexec,nosuid,nodev 0 0
none /sys/fs/cgroup tmpfs rw 0 0
none /sys/fs/fuse/connections fusectl rw 0 0
none /sys/kernel/debug debugfs rw 0 0
none /sys/kernel/security securityfs rw 0 0
udev /dev devtmpfs rw,mode=0755 0 0
devpts /dev/pts devpts rw,noexec,nosuid,gid=5,mode=0620 0 0
tmpfs /run tmpfs rw,noexec,nosuid,size=10%,mode=0755 0 0
none /run/lock tmpfs rw,noexec,nosuid,nodev,size=5242880 0 0
none /run/shm tmpfs rw,nosuid,nodev 0 0
none /run/user tmpfs rw,noexec,nosuid,nodev,size=104857600,mode=0755 0 0
none /sys/fs/pstore pstore rw 0 0
tracefs /var/lib/ureadahead/debugfs/tracing tracefs rw,relatime 0 0
/dev/sda1 /boot ext2 rw 0 0
systemd /sys/fs/cgroup/systemd cgroup rw,noexec,nosuid,nodev,none,name=systemd 0 0
/dev/sdb1 /disk/0 xfs rw,noatime,nodiratime,inode64 0 0
/dev/sdc1 /disk/1 xfs rw,noatime,nodiratime,inode64 0 0
/dev/sdd1 /disk/2 xfs rw,noatime,nodiratime,inode64 0 0
/dev/sde1 /disk/3 xfs rw,noatime,nodiratime,inode64 0 0```